java-common-protos/.github/release-please.yml

```diff
@@ -1,20 +0,0 @@
-bumpMinorPreMajor: true
-handleGHRelease: true
-releaseType: java-yoshi
-branches:
-  - bumpMinorPreMajor: true
-    handleGHRelease: true
-    releaseType: java-yoshi
-    branch: java7
-  - bumpMinorPreMajor: true
-    handleGHRelease: true
-    releaseType: java-backport
-    branch: 2.6.x
-  - bumpMinorPreMajor: true
-    handleGHRelease: true
-    releaseType: java-backport
-    branch: 2.8.x
-  - bumpMinorPreMajor: true
-    handleGHRelease: true
-    releaseType: java-backport
-    branch: 2.10.x
```

Comment:
How have we decided to handle these LTS branches for `java-common-protos`?

Reply:
Same as `api-common-java`, we will keep using the LTS branches in the old `java-common-protos` repo.